RELEASE_PROCESS.md

```diff
@@ -44,25 +44,42 @@ In order to release, you will need the following accounts/permissions:
-  *specifically* test that the issue in question is fixed.
-
-If one of these tests has failed, you cannot ship a release until it's fixed.
-Note, no changes other than CHANGELOG/RELEASE_NOTES changes should
-land on master between testing and releasing since we typically tag
-HEAD of master. If something large does land on master, the release
-tag you create should point specifically at the build that you tested.
-The git SHA of the build you are testing can be found in
-`/opt/opscode/version-manifest.json`.
+Every merge to chef-server master must be built, and this build must be tested with the 'umbrella' automated integration test pipeline: https://buildkite.com/chef/chef-umbrella-master-chef-server.
+The integration test run for the tag being shipped must be successful.
+
+- Click on 'New Build'.
+- Leave 'Branch' set to 'master'.
+- Click on 'Options' to expand the 'Environment Variables' field.
+- Enter `UPGRADE_VERSION=<build record>` into the 'Options | Environment Variables' field, where `<build record>` is the version number string of the omnibus/adhoc build you wish to test (e.g. 13.1.64+20201234567890).
+- Optionally, fill-in the 'Message' field with something descriptive, e.g. the version number string of the build record.
+- Click on 'Create Build'.
+
+Currently, the Integration Test Pipeline does not perform a test login to Chef Manage, so this should be done manually by picking a representative Azure scenario and a representative AWS scenario:
+
+- NOTE: NOT SURE IF THESE INSTRUCTIONS ARE CORRECT.
+- Ensure that your chosen scenario installs the Chef Manage add-on.  This is normally done by default, but can be explicitly enabled by setting the option `ENABLE_ADDON_CHEF_MANAGE=true`.
+- Obtain the DNS name of the ephemeral machine by observing the output of the boot-up.  A sample output is shown below:
+```
+null_resource.chef_server_config: Provisioning with 'remote-exec'...
+null_resource.chef_server_config (remote-exec): Connecting to remote host via SSH...
+null_resource.chef_server_config (remote-exec):   Host: ec2-34-212-122-231.us-west-2.compute.amazonaws.com
+null_resource.chef_server_config (remote-exec):   User: ec2-user
+null_resource.chef_server_config (remote-exec):   Password: false
+null_resource.chef_server_config (remote-exec):   Private key: false
+null_resource.chef_server_config (remote-exec):   Certificate: false
+null_resource.chef_server_config (remote-exec):   SSH Agent: true
+null_resource.chef_server_config (remote-exec):   Checking Host Key: false
+null_resource.chef_server_config (remote-exec): Connected!
+null_resource.chef_server_config (remote-exec): echo -e '
+null_resource.chef_server_config (remote-exec): BEGIN INSTALL CHEF SERVER
+```
+- Hit the server via `http://<hostname>` where hostname is the DNS name of the emphemeral machine obtained in an earlier step. [QUESTION: how is this supposed to be done? just use a web browser and it works?]
```

Comment:
yeah, logging from the web browser into manage should be a sufficient test for now.

Reply:
Ok, assuming the above instructions are correct.  I will also run through this entire scenario to see if everything works correctly.